src/init_with_composer.php

```diff
@@ -53,6 +53,7 @@
         '/vendor/sebastian/diff/' => '/vendor/sebastian/diff/',
         '/vendor/nikic/php-parser/' => '/vendor/nikic/php-parser/',
         '/vendor/symfony/polyfill' => '/vendor/symfony/polyfill',
+        '/vendor/guzzlehttp/' => '/vendor/guzzlehttp/',
```

Comment:
There is `SoftMocks:: addIgnoreSubPath()` method that you need to use instead. The list provided here should only ignore things that SoftMocks depends on in order to operate (that cannot be mocked) and guzzlehttp does not look like one of them.